Automattic/jetpack
# Jetpack Sync: Ensure we always sync heartbeat data and active plugins no matter the consumer config

Jetpack Sync: Ensure we always sync heartbeat data and active plugins no matter the consumer config

## Proposed changes:

* The `active_plugins` options has been added to the list of options we always sync no matter the consumer configuration
* The Sync Stats module, responsible for sending heartbeat data to WPCOM, has been added to the list of modules we always sync no matter the consumer configuration

### Other information:


## Jetpack product discussion


p9dueE-8gv-p2#comment-10415

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Activate only Jetpack Protect since it's one of the Sync consumers that is using a [minimum configuration for Sync](https://github.com/Automattic/jetpack/blob/04d14fe60c5e1f8839795c16b4e0631d3adee166/projects/plugins/protect/src/class-jetpack-protect.php#L88)
* Without the patch, the heartbeat data should never get send
* With the patch the heartbeat data should get sent

In order to remove the restriction of syncing the heartbeat data once per day, you can checkout the corresponding instructions here: D152955-code
